### PR TITLE
Update ManagedNetwork

### PR DESCRIPTION
**Pull request by yanfa317**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by yanfa317** [comment]
Move location to resource so that policy/group will contain it.

**Comment by yanfa317** [comment]
@azuresdkci  It is patch from MNC team, and we are moving location from trackresource to resource

**Comment by yanfa317** [comment]
@MyronFanQiu Hi, can you help us review it? It blocks our jobs. Thanks!

**Comment by yanfa317** [comment]
Add python md

**Comment by MyronFanQiu** [comment]
@zikalino Hi Zim. Could you please check the python.md? I just saw the `multi-api` in the md file. @yanfa317 Maybe you should also modify the readme.md as well. Zim is the python SDK owner.

**Comment by yanfa317** [comment]
@MyronFanQiu I added the python md since we need to create python sdk. I copied and rewrite it from other models. Should I remove the multi-api part? Thanks

**Comment by yanfa317** [comment]
@MyronFanQiu@zikalino I have remove the multi api part. Can you help us review it again? Thanks!

File: specification/managednetwork/resource-manager/readme.python.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Tag: package-2019-06-01-preview and python
+
+These settings apply only when `--tag=package-2019-06-01-preview --python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+``` yaml $(tag) == 'package-2019-06-01-preview' && $(python)
+python:
+  namespace: azure.mgmt.managednetwork.v2019_06_01_pre
+  output-folder: $(python-sdks-folder)/managednetwork/azure-mgmt-managednetwork/azure/mgmt/Microsoft.ManagedNetwork/v2019_06_01_pre
```
Comment (MyronFanQiu) [inline specification/managednetwork/resource-manager/readme.python.md]: v2019_06_01_preview?

**Comment by MyronFanQiu** [comment]
@yanfa317 Hi. @zikalino will help you to generate Python SDK. So it's will much helpful to let him review this PR before you send an SDK request to him. Multi-API is not a problem. I just don't have enough knowledge for this part. Zim will go through this readme.python.md anyway eventually. @zikalino Could you please take a look?

File: specification/managednetwork/resource-manager/readme.python.md (hunk 1 above)
Comment (yanfa317) [reply specification/managednetwork/resource-manager/readme.python.md]: Yes, it is our api version. Is there any problem here?

**Comment by yanfa317** [comment]
Thanks @MyronFanQiu @zikalino Can you explain what is multi api? Do we need it? Also, can you help us review it early? It blocks a lot of our works. Thanks so much!

File: specification/managednetwork/resource-manager/readme.python.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  clear-output-folder: true
+  no-namespace-folders: true
+```
+
+### Python multi-api
+
+Generate all API versions currently shipped for this package
+
+```yaml $(python) && $(multiapi)
+batch:
```
Comment (zikalino) [inline specification/managednetwork/resource-manager/readme.python.md]: for single api there shouldn't be batch section

**Review by zikalino — APPROVED** [review]

**Review by zikalino — APPROVED** [review]

**Pull request merged**